capstone-app/src/app/modules/photos/components/image-container/image-container.component.ts

```diff
@@ -0,0 +1,56 @@
+import { MockPhotoFetcher } from './../../services/mock-photo-fetcher.service';
+import { Photo } from './../../photo-template';
+import { PhotoFetcher } from './../../services/photo-fetcher.service';
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-image-container',
+  templateUrl: './image-container.component.html',
+  styleUrls: ['./image-container.component.scss'],
+})
+export class ImageContainerComponent implements OnInit {
+  photos: Photo[] = [];
+
+  constructor(
+    private photosService: PhotoFetcher,
+    private mockPhotosService: MockPhotoFetcher
+  ) {}
+
+  ngOnInit() {
+    this.getMockPhotos();
+  }
+
+  getPhotos() {
+    this.photosService.getPhotos().subscribe((results) => {
```

Comment:
This doesn't need to be in this pr, but eventually we should have some handling either for failed requests or when 0 images are returned. Perhaps a message could display that says "No images were returned" or "the search for images failed". If you add those, tests should be written for those code paths as well.

If you want to do those in a subsequent pr, add a TODO.

```
// TODO(billyporter): Add handling for requests that fail and requests that return 0 results.